app/views/home/index.html.haml

```diff
@@ -2,4 +2,4 @@
 
 = link_to "Sign up", new_user_registration_path
 = link_to "Sign in", new_user_session_path
-
+= link_to "Log out", destroy_user_session_path, method: :delete
```

Comment:
Do you need to state the method here?
Does it work without it?

Reply:
We added it because it didn't work without of some reason